seerep-hdf5/seerep-hdf5-pb/test/pb-write-load-test.cpp

```diff
@@ -0,0 +1,327 @@
+#include <gtest/gtest.h>
+#include <seerep-core/core.h>
```

Comment:
The include of the core is unnecessary, because you created the HDF5 file by hand without using the core. You can remove it as well as the dependencies in CMakeList and in the package.xml